.github/workflows/main.yaml

```diff
@@ -19,3 +19,9 @@ jobs:
       - uses: Swatinem/rust-cache@v2
       - run: cargo build --locked --verbose
       - run: cargo test --locked
+  docker-build:
+    name: Build the Docker image
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - run: docker build . --file Dockerfile --tag pod-director:$(date +%s)
```

Comment:
Question. Would we like to be _petty_ and adopt the "Containerfile" and "container image" terms instead?

Ubuntu uses it in its documentation: https://manpages.ubuntu.com/manpages/jammy/en/man5/containers-dockerfile.5.html

I'll let you decide

Reply:
Yes, we would! 

Although 'Dockerfile' is the industry standard, using a more generic name would be better.

Changed in [3368c8c](https://github.com/Angelin01/pod-director/pull/16/commits/3368c8c0ed2ab7fb34dfa2a4612f0dd1668bfcec)